src/overlays/actors/ovl_En_Test2/z_en_test2.c

```diff
@@ -5,15 +5,28 @@
  */
 
 #include "z_en_test2.h"
+#include "objects/object_dekucity_ana_obj/object_dekucity_ana_obj.h"
+#include "objects/object_sichitai_obj/object_sichitai_obj.h"
+#include "objects/object_yukimura_obj/object_yukimura_obj.h"
+#include "objects/object_hakugin_obj/object_hakugin_obj.h"
+#include "objects/object_meganeana_obj/object_meganeana_obj.h"
+#include "objects/object_haka_obj/object_haka_obj.h"
 
 #define FLAGS (ACTOR_FLAG_10 | ACTOR_FLAG_80)
 
 #define THIS ((EnTest2*)thisx)
 
 void EnTest2_Init(Actor* thisx, PlayState* play);
 void EnTest2_Update(Actor* thisx, PlayState* play);
+void EnTest2_SetActorFlags(Actor* thisx, PlayState* play);
+void EnTest2_Draw(Actor* thisx, PlayState* play);
+
+typedef struct EnTest2DisplayObjs {
+    Gfx* DList1;
+    Gfx* DList2;
```

Comment:
Should be camelCase. Also don't need to name the struct at the beginning for something that's only used once like this.